### PR TITLE
Add SQL and QueryWorkbench to 2.7.0 manifest

### PR DESCRIPTION
**Pull request by penghuo**
### Description
Add SQL and QueryWorkbench to 2.7.0 manifest

### Issues Resolved
* QueryWorkbench, https://github.com/opensearch-project/dashboards-query-workbench/issues/56
* SQL, https://github.com/opensearch-project/sql/issues/1492

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.7.0/opensearch-2.7.0.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ components:
       - gradle:dependencies:opensearch.version
   - name: sql
     repository: https://github.com/opensearch-project/sql.git
-    ref: 2.x
+    ref: 2.7
```
Comment (gaiksaya) [inline manifests/2.7.0/opensearch-2.7.0.yml:126]: Please enclose 2.7 in quotes.
```suggestion
    ref: '2.7'
```

File: manifests/2.7.0/opensearch-dashboards-2.7.0.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ components:
     ref: 2.x
   - name: queryWorkbenchDashboards
     repository: https://github.com/opensearch-project/dashboards-query-workbench.git
-    ref: 2.x
+    ref: 2.7
```
Comment (gaiksaya) [inline manifests/2.7.0/opensearch-dashboards-2.7.0.yml:25]:
```suggestion
    ref: '2.7'
```

File: manifests/2.7.0/opensearch-2.7.0.yml (hunk 1 above)
Comment (penghuo) [reply manifests/2.7.0/opensearch-2.7.0.yml]: done.

File: manifests/2.7.0/opensearch-dashboards-2.7.0.yml (hunk 2 above)
Comment (penghuo) [reply manifests/2.7.0/opensearch-dashboards-2.7.0.yml]: done.

**Review by gaiksaya — APPROVED** [review]

**Review by zelinh — APPROVED** [review]

**Pull request merged**